src/popover/popover.test.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+var $ = require('jquery');
+var angular = require('angular');
+require('angular-mocks');
+require('./index.js');
+
+describe('Directive: buildium.angular-elements.popover', function () {
+    var elem,
+        $compile,
+        scope,
+        popoverCtrl,
+        mockSubmenuSvc,
+        mockPopUps,
+        mockFilter;
+
+    beforeEach(angular.mock.module('buildium.angular-elements.popover', function ($provide) {
+        mockSubmenuSvc = {
+            display : function display() {},

```

Comment:
we create these noops and then spy on them later, but we can do that in one step if we set this to `jasmine.createSpy('display')` here, for example